src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdaterTest.java

```diff
@@ -103,7 +104,7 @@ void updateCase_should_if_no_warnings() {
         Optional<ErrorsAndWarnings> res = ccdCaseUpdater.updateCase(
             exceptionRecord,
             configItem,
-            true,
+            false,
```

Comment:
can you explain?

Reply:
This is actually `updateCase_should_pass_if_no_warnings` method moved here instead of the existing one with unclear method name.